### DISCUSSION TITLE
Coin Cell Model

### DISCUSSION THREAD

**Discussion by kabravenkatesh** [pybamm-team/PyBaMM#3152] [Q&A]
Hello Everyone,

I am modeling the behavior of a coin cell using the 'Ai2020' dataset and study the EIS response of the cell. 

Here are the parameter for the cell:
```python3
## cell parameters ##
params = pybamm.ParameterValues('Ai2020')
params['Cell cooling surface area [m2]'] = 0.000628319   #top and bottom area of coin cell
params['Cell volume [m3]'] = 1.00531E-06        #cylinder volume  
params['Nominal cell capacity [A.h]'] = 0.045   #45 mAh capacity
params['Number of electrodes connected in parallel to make a cell'] = 5 

params['Negative electrode thickness [m]'] = 7.69141E-05   #5*capacity of single electrode = 45 mAh
params['Positive electrode thickness [m]'] = 6.21138E-05
params['Electrode height [m]'] = 0.017724539    #sqrt(Coin cell area of cross section)
params['Electrode width [m]'] = 0.017724539     #sqrt(Coin cell area of cross section)

##building model ##
model = pybamm.lithium_ion.DFN(build=False,   options =  {
      "SEI": "reaction limited", 'surface form':'differential'
    }
)
model.build_model()

#current signal

def my_fun(fSig):
    def custom_cur(t):
        A = 5e-3        #5 mA for cell -> 1 mA for layer
        return A * pybamm.sin(2 * np.pi * fSig * t)       #excitation signal
    return custom_cur

fSig = 100 #Hz
Fs = 10*fSig  ##sampling frequency
params['Current function [A]'] = my_fun(fSig)
TSig = 1/fSig;   #timeperiod of signal
ts = 0.0
te = 2*TSig - 1/Fs #makes sure that fSig is the multiple of fbin. No spectral leakage

tsim = np.linspace(ts, te, int((te - ts)*Fs + 1))
sim = pybamm.Simulation(model, parameter_values = params)
results = sim.solve(t_eval=tsim)
````

My main issue is that when I compare with experimental data, the impedance response of model is larger by a factor of ~5.  Is there any parameter which requires to be tuned or changed in order to make the model for the coin cell. 
One more issue: 'Terminal voltage [V]' does not have the effect of contact resistance of the cell. Is there any other model variable which outputs the V-I*R drop for the cell? 

Thanks a lot for the help!

**Comment by brosaplanella** [comment]
Note that the parameters in Ai2020 are specific for that cell, so if you change the cell you want to test you will have to consider readjusting the whole parameter set. I notice you you have 5 electrodes in parallel, could that be related to your factor of 5?

From version 23.1 onwards, you can pass a `Contact resistance [Ohm]` to the model which is included in the voltage.

**Reply by kabravenkatesh** [reply]
I am trying to figure out if there is any particular parameter which I am not adjusting resulting in an unphysical pouch cell design (Since Ai2020 is based on the pouch cell parameters) instead of the coin cell.  
The Ai2020 parameter set has 34 electrode layers initially for capacity of 2.28 Ah. I back calculated the number of layers required to enable a capacity of 45 mAh (~5 layers). Does it makes sense? In general the smaller the cell the smaller should be the impedance?

**Reply by brosaplanella** [reply]
But you are also making the electrode 5 times thicker, aren't you?

**Reply by kabravenkatesh** [reply]
I thought number of electrodes is the number of electrodes being simulated in parallel. Since a given Li-ion cell consists of multiple parallel electrode layers. For example the default value of the params['Number of electrodes connected in parallel to make a cell']  = 34 for Ai2020 dataset. So earlier the resistance is R/N where N is being changed. But the issue is a coincell only has between 5-10 layers of electrode unlike the cylindrical cell modeled in the Ai2020 dataset.
@rtimms @brosaplanella @tinosulzer

**Reply by kabravenkatesh** [reply]
Hi all, am I understanding the values of parameters correctly ?

**Reply by brosaplanella** [reply]
Yes, but if I understand you correctly you made the electrode 5 times thicker AND simulated 5 cells in parallel, which would yield a capacity 25 times larger than the original one.
